ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -71,8 +71,17 @@
           data:
             .dockerconfigjson: "{{ lookup('file', operator_pipeline_quay_token, rstrip=False) | b64encode }}"
     - name: Link secret with pipeline service account (for imagestream operations)
-      ansible.builtin.shell: |
-        oc secret link pipeline registry-dockerconfig-secret
+      k8s:
+        state: present
+        force: yes
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: v1
+          kind: ServiceAccount
+          metadata:
+            name: pipeline
+          secrets:
+            - name: registry-dockerconfig-secret
```

Comment:
Doesn't this override previous secrets for the SA?  Since the SA comes from a pipeline operator it can have predefined secrets.

Reply:
It doesn't, I tested it

Reply:
ok, then Is fine.